README.md

```diff
@@ -15,12 +15,12 @@ rovercode runs on any single-board-computer supported by the [Adafruit Python GP
 ## Easy Setup
 First, on your robot:
 ```
-$ git clone https://git.io/rovercode.git
+$ git clone --recursive https://github.com/aninternetof/rovercode.git
+$ sudo ./setup.sh
+$ sudo ./start.sh
```

Comment:
Do you want to mention something here about how start.sh would need to be run each power cycle but not the others?